src/eventkit.cs

```diff
@@ -611,6 +616,7 @@ interface EKEventStore {
 #if MONOMAC
 		[Deprecated (PlatformName.MacOSX, 10, 9)]
 		[Export ("initWithAccessToEntityTypes:")]
+		[NoMacCatalyst]
```

Comment:
@rolfbjarne  @spouliot @tj-devel709  if that NoMacCatalust is inside the MONOMAC, does it make any effect? Should it be with the other attrs?

Reply:
No, it doesn't not have effect inside an `#if MONOMAC`

Reply:
@tj-devel709  that means that the attr has to be outside the MONOMAC ;) (I suspected so)